### PR TITLE
Added cross platform scripts to extension

### PR DESCRIPTION
**Pull request by tdusnoki**
IoT.js-VSCode-DCO-1.0-Signed-off-by: Tibor Dusnoki tdusnoki@inf.u-szeged.hu

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -70,47 +71,37 @@ const provideInitialConfigurations = (): string => {
...
+  const rm = /^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$/;
+  await getPath();
+  if (tizenStudioPath && address && address !== 'localhost') {
+    if (rm.test(address)) {
+      buildProject(tizenStudioPath, address);
+    } else {
+      vscode.window.showErrorMessage('Please enter a valid ip address');
     }
+  } else {
+    vscode.window.showErrorMessage('Please enter remote address instead of localhost');
```
Comment (daeyeon) [inline src/extension.ts:101]: This message looks wrong in the case that `tizenStudioPath` is null.

Comment (tdusnoki) [reply src/extension.ts:101]: Fixed error message, thanks for pointing it out!

**Review by daeyeon — APPROVED** [review]
LGTM

**Review by LaszloLango — APPROVED** [review]
LGTM

**Pull request merged**